ext/travis/test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "$TRAVIS_OS_NAME" = linux ]; then
+    uid="$(id -u)"
+    gid="$(id -g)"
+    tinyfs="$(pwd)/tinyfs"
+
+    sudo -i dd if=/dev/zero of=tinyfs.img bs=10MB count=1
+    sudo -i chmod 600 tinyfs.img
+    sudo -i mke2fs -F tinyfs.img
+    sudo -i mkdir "$tinyfs"
+    sudo -i mount -oloop tinyfs.img "$tinyfs"
+    sudo -i chown "$uid:$gid" "$tinyfs"
```

Comment:
@waynr thanks, and right -- I'd be very hesitant to do anything like this on a shared host.  We'll wait until it's safe.